includes/Products/Sync/Background.php

```diff
@@ -102,6 +102,30 @@ public function process_job( $job, $items_per_batch = null ) {
 		return $job;
 	}
 
+	/**
+	 * Handles job completion.
+	 *
+	 * @since 2.6.0
+	 *
+	 * @param \stdClass|object|string $job Job instance or ID.
+	 * @return \stdClass|object|false on failure.
+	 */
+	public function complete_job( $job ) {
+
+		if ( is_string( $job ) ) {
+			$job = $this->get_job( $job );
+		}
+
+		if ( ! $job ) {
+			return false;
+		}
+
+		$job->status       = 'completed';
+		$job->completed_at = current_time( 'mysql' );
```

Comment:
This method overrides [base `complete_job` from the SkyVerge framework](https://github.com/skyverge/wc-plugin-framework/blob/master/woocommerce/utilities/class-sv-wp-background-job-handler.php#L742), and changes the behaviour, by calling another base class lifecycle method - `delete_job`.

That seems risky to me - perhaps there's a cleaner supported way to manage the options that accumulate over time.